src/components/transcript/index.tsx

```diff
@@ -60,6 +65,58 @@ export type sideBarContentUpdateParams<T, K> = {
   name: K;
 };
 
+const getTranscriptContent = (content: TranscriptContent) => {
+  // eslint-disable-next-line no-unused-vars
+  const { body, media, transcript_by, ...pickedContent } = content;
+  const {
+    speakers,
+    categories,
+    tags,
+    title = "",
+    loc = "",
+    date,
+    ...restContent
+  } = pickedContent;
```

Comment:
I am destructing the first values as a way to omit them, because I don't need them.

Doing this this other suggested way would raise an eslint warning of unused vars. So I would have to suppress like so:

```ts
  const {
    // eslint-disable-next-line no-unused-vars
    body,
    // eslint-disable-next-line no-unused-vars
    media,
    // eslint-disable-next-line no-unused-vars
    transcript_by,
    speakers,
    categories,
    tags,
    title = "",
    loc = "",
    date,
    ...restContent
  } = content;
```

Which is more verbose

Reply:
understood! Can you add a comment there explaining that we don't need them?
Also, based on the excluded content, it seems that `getTranscriptMetadata` or `extractTranscriptMetadata` is a better name for this function.
Then you can also rename `pickedContent` to `metadata` and `restContent` to `arbitraryMetadata`.

I believe that those renames will help with readability of the logic there.